src/figuresClasses.ts

```diff
@@ -1,11 +1,98 @@
-export interface Figure {}
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  color: Color;
+  shape: Shape;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
```

Comment:
[CODE KNOWLEDGE] - You're declaring constructor parameters with an access modifier (public). TypeScript implicitly creates a property on the class with the same name as the parameter and assigns the parameter value to that property. So, you don't need to manually assign values to these properties. Refer to the checklist for an example.